judo-ui-react/src/main/resources/actor/src/fragments/table/row-actions.fragment.hbs

```diff
@@ -32,7 +32,7 @@ const {{ table.dataElement.name }}RowActions: TableRowAction<{{ classDataName ta
                 label: t('{{ getTranslationKeyForAction action }}', { defaultValue: '{{ action.label }}' }) as string,
                 icon: <MdiIcon path="{{ action.icon.name }}" />,
                 action: async (row: {{ classDataName table.dataElement.target 'Stored' }}) => {{ actionFunctionName action }}({{# if action.operation.isMapped }}row, {{/ if }}() => fetchOwnerData()),
-                disabled: (row: {{ classDataName table.dataElement.target 'Stored' }}) => editMode,
+                disabled: (row: {{ classDataName table.dataElement.target 'Stored' }}) => {{# if action.enabledBy }}!row.{{ action.enabledBy.name }} ||{{/ if }}{{# if action.hiddenBy }} row.{{ action.hiddenBy.name }} ||{{/ if }} editMode,
```

Comment:
I currently do not recommend disabling the operation if it's hidden because there isn't any other built-in method to add actions to table rows aside from adding them to the view screen and then hiding them.

Reply:
[efc480f](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/196/commits/efc480f8802ce157f9024a58d3492a1e9087cd74)